ports/powerputtlegend/port.json

```diff
@@ -0,0 +1,28 @@
+{
+  "version": 3,
+  "name": "powerputtlegend.zip",
+  "items": [
+    "powerputtlegend/",
+    "PowerPutt Legend.sh"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "PowerPutt Legend",
+    "porter": [
+      "coolz"
+    ],
+    "desc": "PowerPutt Legend is a unique minigolf game with RPG elements, meant to look and feel like it's straight out of the NES. Slay monsters, level up, and unlock a series of items to improve your strategy.",
+    "inst": "Ready to run! Thanks to Team Bugulon granting us permission to distribute.",
+    "genres": [
+      "adventure"
+    ],
+    "image": null,
+    "rtr": true,
```

Comment:
rtr:true means gamedata is included.
If that is the case a license needs also to be provided.